### PR TITLE
Parameterize JAVA_OPTS

### PR DESCRIPTION
**Pull request by Isakgicu**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -135,6 +135,12 @@ variable "time_limit_ms" {
   type        = number
 }
 
+variable "java_opts" {
+  description = "Custom JAVA Options"
```
Comment (andrein) [inline variables.tf]:
```suggestion
  description = "Custom Java Options"
```

File: variables.tf [hunk 2]
```diff
@@ -135,6 +135,12 @@ variable "time_limit_ms" {
...
+  default     = "-Dorg.slf4j.simpleLogger.defaultLogLevel=info -Dcom.amazonaws.sdk.disableCbor"
```
Comment (andrein) [inline variables.tf]:
```suggestion
  default     = ""
```

File: templates/user-data.sh.tmpl [hunk 3]
```diff
@@ -41,7 +41,7 @@ sudo docker run \
 %{ endif ~}
   -v $${CONFIG_DIR}:/snowplow/config \
   -p ${port}:${port} \
-  -e 'JAVA_OPTS=-Dorg.slf4j.simpleLogger.defaultLogLevel=info -Dcom.amazonaws.sdk.disableCbor' \
+  -e 'JAVA_OPTS=${java_opts}' \
```
Comment (andrein) [inline templates/user-data.sh.tmpl]: `-Dcom.amazonaws.sdk.disableCbor` [looks pretty important here](https://github.com/aws/aws-sdk-java/issues/2493), maybe we shouldn't risk losing it? If users want to change the loglevel they could just set `-Dorg.slf4j.simpleLogger.defaultLogLevel=somethingelse` and it should override the first option. @archeus, thoughts?

```suggestion
  -e 'JAVA_OPTS=-Dorg.slf4j.simpleLogger.defaultLogLevel=info -Dcom.amazonaws.sdk.disableCbor ${java_opts}' \
```

Comment (archeus) [reply templates/user-data.sh.tmpl]: yes @andrein, I agree, we should keep `-Dcom.amazonaws.sdk.disableCbor` here so we don't accidentaly lose it when adding other java opts

**Pull request merged**